src/PageLayout/Notification/Details/RequestSummary.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import RequestTime from './RequestTime';
+import truck from '../../../icons/green_truck.svg';
+import Map from '../../../Map/Map.js';
+
+class RequestSummary extends React.Component {
+    render() {
+        return (
+            <div className="modal-wrapper">
+                <div className="modal-header-flex">
+                    <img className="icon" src={truck} alt="icon" />
+                    <h1 id="modal-step1">{this.props.title}</h1>
+                </div>
+                <div className="details-scroll">
+                    <div className="pickup-details">
+                        <RequestTime request={this.props.details} />
+                    </div>
+                    <div className="flex">
+                        <div className="donating-agency">
+                            <h2 id="organizationName">Donating Agency</h2>
+                            <p className="donating-recieving" id="donatingAgency">{this.props.details.daInfo.name}</p>
+                            <p className="donating-recieving" id="donatingContactName">{this.props.details.daInfo.primaryContact.name}</p>
+                            <p className="donating-recieving" id="donatingContactPhone">{this.props.details.daInfo.primaryContact.phone}</p>
+                            <p className="donating-recieving" id="donatingContactEmail">{this.props.details.daInfo.primaryContact.email}</p>
+                        </div>
+                        <div className="notification-map">
+                            <Map 
+                                address={this.props.details.daInfo.address} 
+                                height={'150px'}
+                                width={'300px'}
+                                marginRight={'20px'}
+                                marginTop={'0px'}
+                                marginBottom={'0px'}
+                                marginLeft={'10px'}
+                                className="map"></Map>
+                        </div>
+                    </div>
+                    { /* View for DG*/
```

Comment:
(now that you know it's not exactly "view for DG", if you wanna remove this comment :) )